CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+# Contributing to Canvas
+
+You may read our contribution guidelines
+[here](https://workday.github.io/canvas-kit/?path=/story/welcome-contributing--page).
```

Comment:
Is this the right link? I was confused to realize that there is a `/getting-started/for-developers/` path for other links in this file.

Reply:
This is the link I would expect.